inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -390,6 +392,16 @@ void MKLDNNGraph::InitOptimalPrimitiveDescriptors() {
     }
 }
 
+void MKLDNNGraph::SplitNodes() {
+    OV_ITT_SCOPE(FIRST_INFERENCE, itt::domains::MKLDNN_LT, "MKLDNNGraph::SplitNodes");
+    for (auto& graphNode : graphNodes) {
+        if (graphNode->isConstant())
+            constantGraphNodes.emplace_back(graphNode);
+        else
+            inconstantGraphNodes.emplace_back(graphNode);
```

Comment:
M.b. rename to `mutableGraphNodes` or `regularGraphNodes`.  What do you think?

Reply:
I think `constantGraphNodes` is clear but I agree that it is worth renaming `inconstantGraphNodes`